techniques/aria/ARIA12.html

```diff
@@ -35,39 +43,53 @@ <h3>Additional heading levels</h3>
    ...
 &lt;div role="heading" aria-level="7"&gt;Jonagold/div&gt;
```

Comment:
There's a small typo in this example with the lack of `<` in closing tag (it's been err... _not_ there since WCAG 2.0)

Reply:
```suggestion
&lt;div role="heading" aria-level="7"&gt;Jonagold&lt;/div&gt;
```

Reply:
I think we made that fix at just about the exact same time. The "commit suggestion" button is grayed out, so I can't apply your fix, so I'm going to resolve it.